core/src/ecs/Entity.ts

```diff
@@ -54,12 +55,13 @@ export const Entity = <T extends ComponentTypes>(protoEntity: ProtoEntity<T>): E
 export const deserializeEntity = (entity: ProtoEntity, serializedEntity: SerializedEntity): void => {
 
   // add new components if necessary
-  Object.keys(serializedEntity).forEach((type) => {
+  Object.entries(serializedEntity).forEach(([type, data]) => {
     if (!(type in entity.components)) {
-      console.log(`adding component ${type}`);
       if (type === "controlling") {
         console.log("adding controlling");
         entity.components.controlling = new Controlling();
+      } else {
+        console.error(`failed to add component: ${type}`);
```

Comment:
saw a one-off weird error from the rapier WASM runtime failing to create a polyline collider